src/screens/ImportWalletChooseTypeScreen.tsx

```diff
@@ -60,26 +55,35 @@ export class ImportWalletChooseTypeScreen extends React.PureComponent<Props, Sta
           <Text style={styles.title}>{i18n.wallets.importWallet.title}</Text>
           <Text style={styles.subtitle}>{i18n.wallets.importWallet.chooseTypeDescription}</Text>
         </View>
-        <RadioGroup color={palette.secondary} onSelect={this.onSelect} selectedIndex={this.state.selectedIndex}>
-          <RadioButton style={styles.radioButton} value={WalletTypes[0]}>
-            <View style={styles.radioButtonContent}>
-              <Text style={styles.radioButtonTitle}>{HDSegwitP2SHArWallet.typeReadable}</Text>
-              <Text style={styles.radioButtonSubtitle}>{i18n.wallets.add.ar}</Text>
-            </View>
-          </RadioButton>
-          <RadioButton style={styles.radioButton} value={WalletTypes[1]}>
-            <View style={styles.radioButtonContent}>
-              <Text style={styles.radioButtonTitle}>{HDSegwitP2SHAirWallet.typeReadable}</Text>
-              <Text style={styles.radioButtonSubtitle}>{i18n.wallets.add.air}</Text>
-            </View>
-          </RadioButton>
-          <RadioButton style={styles.radioButton} value={WalletTypes[2]}>
-            <View style={styles.radioButtonContent}>
-              <Text style={styles.radioButtonTitle}>{i18n.wallets.add.legacyTitle}</Text>
-              <Text style={styles.radioButtonSubtitle}>{i18n.wallets.add.legacy}</Text>
-            </View>
-          </RadioButton>
-        </RadioGroup>
+        <>
+          <RadioButton
+            title={HDSegwitP2SHArWallet.typeReadable}
+            subtitle={i18n.wallets.add.ar}
+            value={'2-Key Vault'}
+            selectedValue={this.state.selectedWalletType}
+            onPress={value => {
+              this.onSelect(value);
+            }}
+          />
+          <RadioButton
+            title={HDSegwitP2SHAirWallet.typeReadable}
+            subtitle={i18n.wallets.add.air}
+            value={'3-Key Vault'}
+            selectedValue={this.state.selectedWalletType}
+            onPress={value => {
+              this.onSelect(value);
+            }}
+          />
+          <RadioButton
+            title={i18n.wallets.add.legacyTitle}
+            subtitle={i18n.wallets.add.legacy}
+            value={'Standard HD P2SH'}
```

Comment:
that's not true, it is import flow for all of the legacy wallets not only for `Standard HD P2SH`

Reply:
I think it's better now